huaweicloud/services/dms/data_source_huaweicloud_dms_kafka_instances.go

```diff
@@ -190,6 +190,11 @@ func DataSourceDmsKafkaInstances() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+						"management_connect_address": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						// Typo, it is only kept in the code, will not be shown in the docs.
 						"manegement_connect_address": {
```

Comment:
it is better to mark it deprecated

Reply:
done

Reply:
it seems that `Deprecated` is still missing, please check it again.

Reply:
fixed